apps/system/init/init.c

```diff
@@ -130,6 +130,9 @@ int preapp_start(int argc, char *argv[])
 #endif
 
 #ifdef CONFIG_TASH
+	/* Initialize the tash */
+	tash_init();
```

Comment:
Oh, I am so sorry, I mean I want moving tash_start, not init. That's my fault.
current
```
int preapp_start()
{
tash_register_cmds();
tash_start();
}
```
suggestion
```
int preapp_start()
{
tash_start();
tash_register_cmds();
}
```